### PR TITLE
Chameleon overhaul

### PR DESCRIPTION
**Pull request by Markolie**
**What does this PR do:**
This pull request ports the chameleon bundle from /tg/, which can be purchased for 2TC. It allows players to fully disguise themselves. They can either disguise individual clothing items, or disguise all of their chameleon clothing at once to a given job. The voice changer is included in this bundle, and can now be disguised as well. Even a cigarette can disguise your voice now!

You can choose to disguise as any job role and individual clothing pieces can pretty much be disguised as any available clothing.

I've also made it so that using a voice changer no longer sets your job to "Unknown" in NTTC, as this could be metagamed pretty hard. It'll always use the assignment on your ID card now, so always combine a chameleon disguise with a Syndicate ID card.

Screenshot: https://i.imgur.com/aLkaMPX.png. All of this clothing is fake, including the PDA.

**Changelog:**
:cl: Markolie
add: Syndicate operatives may now purchase a chameleon bundles (2TC), which allows them to disguise not just their jumpsuit, but all of their equipment. They can change individual clothing as well as the outfit as a whole.
tweak: When using a voice changer, NTTC will no longer default to an "Unknown" job: it'll still use the assignment on your card to prevent meta-gaming. Always combine a voice changer and a Syndicate ID.
/:cl:

### CODE REVIEW

File: code/modules/clothing/masks/gasmask.dm [hunk 1]
```diff
@@ -162,11 +162,6 @@
 	item_state = "sexymime"
 	burn_state = FLAMMABLE
 
-/obj/item/clothing/mask/gas/death_commando
```
Comment (Markolie) [inline code/modules/clothing/masks/gasmask.dm]: This was in the code, but isn't used and doesn't even have a working icon.

**Comment by KasparoVy** [comment]
That's so dope, great port

File: code/modules/clothing/masks/gasmask.dm (hunk 1 above)
Comment (KasparoVy) [reply code/modules/clothing/masks/gasmask.dm]: Uses the same icon as the cyborg visor mask thing, never seen this object used though.

**Comment by variableundefined** [comment]
Chameleon Jumpsuit seems to co-exist at the same time as a buy. Perhaps that should be removed?

**Comment by Markolie** [comment]
Good catch.

**Comment by ID404-NotFound** [comment]
all the sprites are human only. Wearing it as vox has it all wrong.

**Pull request merged**